packages/babel-code-frame/src/index.ts

```diff
@@ -147,7 +147,7 @@ export function codeFrameColumns(
   const highlightedLines = highlighted ? highlight(rawLines, opts) : rawLines;
 
   let frame = highlightedLines
-    .split(NEWLINE)
+    .split(NEWLINE, end)
     .slice(start, end)
```

Comment:
Do we still need `, end` here?

Reply:
It's limit for `.split()`. If we have, supposed, 100 lines and need to grab from 30 to 34. We know the end line it's a 34, so the `.split()` creates the first 34 lines and stops.